sagemaker-datawrangler/joined-dataflow/explore_data.ipynb

```diff
@@ -0,0 +1,281 @@
+{
```

Comment:
<ol><li>Please include a title with H1 heading.</li><li>Writer an introductory passage explaining what this notebook is doing</li><li>Write some descriptions for what you are doing in the cell (i.e. import the necessary packages, etc.).</li><li>In the case that this notebook is part of a sequence of notebooks (i.e. a set of notebooks that should be run in a sequence), please refer to them as well and make it clear in the intro that the notebook is part of a sequence so that the audience is aware. Take a look at the introduction of https://github.com/aws/amazon-sagemaker-examples/blob/main/end_to_end/music_recommendation/01_music_dataprep.flow for an example</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3510/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>